java/pending_code_changes.txt

```diff
@@ -1 +1,8 @@
+Build changes:
+-- OSGi support added to Manifest information when building jar (#1300)
 
+Code changes:
+-- Formatting, naming (LOGGER -> logger) and comment tweaks to follow style
+guide
+-- Removal of unneeded canBeGeocoded method in the
+PhoneNumberToTimeZonesMapper.java, using phoneNumberUtil instead
```

Comment:
Formatting is inconsistent with current release notes (blank line, one dash, space before dash, align wrapped lines). Should be more like:
```nomarkdown
Build changes:
 - OSGi ...
Code changes:
 - Formatting, ...
   guide
 - Removal ...
   PhoneNumberTo...
```